scipy/stats/_entropy.py

```diff
@@ -143,60 +163,173 @@ def differential_entropy(
     .. [2] Crzcgorzewski, P., & Wirczorkowski, R. (1999). Entropy-based
            goodness-of-fit test for exponentiality. Communications in
            Statistics-Theory and Methods, 28(5), 1183-1202.
+    .. [3] Van Es, B. (1992). Estimating functionals related to a density by a
+           class of statistics based on spacings. Scandinavian Journal of
+           Statistics, 61-72.
+    .. [4] Ebrahimi, N., Pflughoeft, K., & Soofi, E. S. (1994). Two measures
+           of sample entropy. Statistics & Probability Letters, 20(3), 225-234.
+    .. [5] Correa, J. C. (1995). A new estimator of entropy. Communications
+           in Statistics-Theory and Methods, 24(10), 2439-2449.
+    .. [6] Noughabi, H. A. (2015). Entropy Estimation Using Numerical Methods.
+           Annals of Data Science, 2(2), 231-241.
+           https://link.springer.com/article/10.1007/s40745-015-0045-9
 
     Examples
     --------
     >>> from scipy.stats import differential_entropy, norm
 
     Entropy of a standard normal distribution:
 
-    >>> # We use high numbered seeds in order to have good entropy in the RNG
     >>> rng = np.random.default_rng()
     >>> values = rng.standard_normal(100)
     >>> differential_entropy(values)
-    1.2723964230956013
+    1.3407817436640392
 
     Compare with the true entropy:
 
     >>> float(norm.entropy())
     1.4189385332046727
 
+    For several sample sizes between 5 and 1000, compare the accuracy of
+    the ``'vasicek'``, ``'van es'``, and ``'ebrahimi'`` methods. Specifically,
+    compare the root mean squared error (over 1000 trials) between the estimate
+    and the true differential entropy of the distribution.
+
+    >>> from scipy import stats
+    >>> import matplotlib.pyplot as plt
+    >>>
+    >>>
+    >>> def rmse(res, expected):
+    ...     '''Root mean squared error'''
+    ...     return np.sqrt(np.mean((res - expected)**2))
+    >>>
+    >>>
+    >>> a, b = np.log10(5), np.log10(1000)
+    >>> ns = np.round(np.logspace(a, b, 10)).astype(int)
+    >>> reps = 1000  # number of repetitions for each sample size
+    >>> expected = stats.expon.entropy()
+    >>>
+    >>> method_errors = {'vasicek': [], 'van es': [], 'ebrahimi': []}
+    >>> for method in method_errors:
+    ...     for n in ns:
+    ...        rvs = stats.expon.rvs(size=(reps, n), random_state=0)
```

Comment:
Leftover. Will not impact any test as for plotting only.
```suggestion
    ...        rvs = stats.expon.rvs(size=(reps, n), random_state=rng)
```